zebra-rpc/src/methods/get_block_template_rpcs.rs

```diff
@@ -568,6 +568,14 @@ where
             let next_block_height =
                 (chain_tip_and_local_time.tip_height + 1).expect("tip is far below Height::MAX");
 
+            tracing::info!(
```

Comment:
Can make all these logs debug or trace level?

They could be too verbose for users if there are a lot of transactions in the mempool or block.

Nitpick: Rust style does not start logs or error messages with a capital letter. I think that's because they are often added to an existing formatted line or sentence.

Reply:
> Can make all these logs debug or trace level?

Yep, I agree these are too verbose for info-level logs, fixed in [`b1cec4f`](https://github.com/ZcashFoundation/zebra/pull/6006/commits/b1cec4fb44d64521b16adca1532ead59a9104d51).